projects/app/test/data/skills.test.ts

```
@@ -313,9 +313,35 @@ void test(skillReviewQueue.name, async () => {
     ]);
   });
+
+  await test(`prioritises due skills with highest value (not most overdue)`, async () => {
+    debugger;
+    const graph = await skillLearningGraph({
+      targetSkills: [hanziWordToEnglish(`分:divide`)],
+      isSkillLearned: () => false,
+    });
+
+    assert.deepEqual(
+      skillReviewQueue({
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)